### DISCUSSION TITLE
Best approach to share albums internally

### DISCUSSION THREAD

**Discussion by Connor0308** [photoprism/photoprism#1292] [Q&A] [answered]
Dear,

I came across PhotoPrism and I really like it sofas. Apart from the UI, I love the idea to have TensorFlow included.
However, what I cannot currently get my fingers on is how to share several albums without having to ask the recipient to use a different password for each album (and using the same password for all albums is obviously not the best idea :-D).

In Adobe LR language I would create a folder containing several albums and share the folder, which would be password protected. Is there any way to achieve anything like this in PhotoPrism currently - and if so, where should I have a closer look at?

Thank you for guiding me,
regards,

Connor

**Comment by graciousgrey** [comment, marked as answer]
At the moment you can share albums using a link see https://docs.photoprism.org/user-guide/share/share/. Those shared albums are not password protected.

Related tickets:
https://github.com/photoprism/photoprism/issues/1162
https://github.com/photoprism/photoprism/issues/557
